addons/binding/org.openhab.binding.bluetooth.ruuvitag/ESH-INF/thing/ruuvitag.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ruuvitag_beacon">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bluegiga" />
+			<bridge-type-ref id="bluez" />
+		</supported-bridge-type-refs>
+
+		<label>RuuviTag SmartBeacon</label>
+		<description>A RuuviTag SmartBeacon</description>
+
+		<channels>
+			<channel id="rssi" typeId="rssi" />
+
+			<channel id="accelerationx" typeId="ruuvitag_accelerationx" />
+			<channel id="accelerationy" typeId="ruuvitag_accelerationy" />
+			<channel id="accelerationz" typeId="ruuvitag_accelerationz" />
+			<channel id="battery_voltage" typeId="ruuvitag_battery_voltage" />
```

Comment:
Not very important. But the preferred notation for channels is: camel case for channels `thisIsAChannel` and hyphen syntax with small caps (`this-is-a-channel`) for channel types.

Reply:
Fixed in ` [7c3a114](/openhab/openhab2-addons/pull/4913/commits/7c3a1147ef12b2d54d3fc92a9962332f9669707d)`

Reply:
Btw, left `accelerationx`, `accelerationy` and `accelerationz` as they were (instead of camel case `accelerationX`), to follow convention set by `blukii` with `tiltx`, `tilty` and `tiltz`.